### ISSUE TITLE
issues about code in agent

### ISSUE THREAD

**Issue by Yang-xiao-zhen** [Leolty/tablellm#2]
Hello,
    in your run_agent.py code, line 60 "with open("output/wtq_agent_wo_norm/result.jsonl", "r") as f:", I didn't see any information about the file. I want to know where the file from, the result of the run_cot? or other result?
Thanks

**Comment by Yang-xiao-zhen** [comment]
Also, I don't know how to implement pyagent+cot combination. Do you use both of them in eval_wtq? Be similar to
checkpoints=["tablellm/output/wtq_dp/result.jsonl", "tablellm/output/wtq_agent/result.jsonl"]
n_times=1
)

**Comment by Leolty** [comment]
Certainly, I'd be happy to help polish the reply for better expression. Let me rephrase the response to make it clearer and more concise:



Hello Zhen,

Thank you for your patience. Let me address your questions:

1. Regarding the line you referenced in `run_agent.py`:
   As explained in our paper, we implement a `norm` process to convert tables into a format preferred by the LLM. However, this process often leaves some tables unchanged. To optimize API usage and reduce costs, we reuse results when possible. If you don't need this, you can directly comment out the related code. Sorry I should have made this step optional. Ideally, there should be a check to see if the file exists.

2. For evaluation code and using the agent without running on specific datasets:
   We have provided several example notebooks in our repository. You can find them here:
   [https://github.com/Leolty/tablellm/tree/main/examples](https://github.com/Leolty/tablellm/tree/main/examples)

These notebooks demonstrate various ways to use the agent and should help you get started.

Please let me know if you need any further clarification or assistance.

**Comment by Leolty** [comment]
Hello Zhen,

Thank you for your patience. Let me address your questions:

1. Regarding the line you referenced in `run_agent.py`:
   As explained in our paper, we implement a `norm` process to convert tables into a format preferred by the LLM. However, this process often leaves some tables unchanged. To optimize API usage and reduce costs, we reuse results when possible. If you don't need this, you can directly comment out the related code. Sorry I should have made this step optional. Ideally, there should be a check to see if the file exists.

2. For evaluation code and using the agent without running on specific datasets:
   We have provided several example notebooks in our repository. You can find them here:
   [https://github.com/Leolty/tablellm/tree/main/examples](https://github.com/Leolty/tablellm/tree/main/examples). And just run them separately and aggregate any files using the `eval_wtq`.

These notebooks demonstrate various ways to use the agent and should help you get started.

Please let me know if you need any further clarification or assistance.
